Unix/configure

```diff
@@ -494,12 +500,14 @@ if [ "$enable_ulinux" = "1" ]; then
     openssl100_cflags=`PKG_CONFIG_PATH=$openssl100_libdir/pkgconfig $pkgconfig --cflags openssl`
     openssl100_libs=`PKG_CONFIG_PATH=$openssl100_libdir/pkgconfig $pkgconfig --libs openssl`
 
+if [ "$disable_ssl_0_9_8" != "1" ]; then
```

Comment:
This should be indented properly to match what's above. Like C++, it's not required, but you should have it for readability.